ibis/omniscidb/operations.py

```diff
@@ -286,6 +289,30 @@ def extract_field_formatter(translator, expr, sql_attr=sql_attr):
     return extract_field_formatter
 
 
+def _extract_field_dow_name(sql_attr):
+    def extract_field_formatter(translator, expr, sql_attr=sql_attr):
```

Comment:
did you enable tests for this?

Reply:
as it is not unsupported anymore the current tests should check it https://github.com/ibis-project/ibis/blob/master/ibis/tests/all/test_temporal.py#L361-L397